sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 12.3.0-beta.1 (Unreleased)
+- Added ability to create service clients anonymously. Anonymous service clients will throw on attempting to create network requests.
```

Comment:
Maybe add "and should only be used to create anonymous container and blob clients." Just in case people wonder "well then what's the point?" lol